### PR TITLE
[global] rename "usage" chapter to "examples" in documentation for multiple modules

### PR DESCRIPTION
**Pull request by duckhawk**
## Description
Rename "usage" chapter to "examples" in documentation for multiple modules

## Why do we need it, and what problem does it solve?
Documentation refactoring for more obvious naming

## What is the expected result?
More obvious documentation structure

## Checklist
- [ ] The code is covered by unit tests.
- [ ] e2e tests passed.
- [x] Documentation updated according to the changes.
- [x] Changes were tested in the Kubernetes cluster manually.

## Changelog entries
```changes
section: global
type: chore
summary: Rename "usage" chapter to "examples" in documentation for multiple modules.
impact_level: low
```

### CODE REVIEW

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @z9r5. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

File: docs/site/.helm/templates/_rewrites.tpl [hunk 1]
```diff
@@ -5,4 +5,33 @@ rewrite ^/ru/terms-of-service\.html /ru/security-policy.html permanent;
 rewrite ^/ru/cookie-policy\.html /ru/security-policy.html permanent;
 rewrite ^/ru/privacy-policy\.html /ru/security-policy.html permanent;
 rewrite ^/en/security-policy\.html /en/privacy-policy.html permanent;
+rewrite ^/(en|ru)/documentation/v1/modules/030-cloud-provider-openstack/usage.html$ /$1/documentation/v1/modules/030-cloud-provider-openstack/examples.html permanent;
```
Comment (apolovov) [inline docs/site/.helm/templates/_rewrites.tpl:8]: don't permanent it :)

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — DISMISSED** [review]

**Comment by deckhouse-BOaTswain** [comment]
Aye, aye, @duckhawk. I've started the workflow for label 'deploy/web/test'...

**Review by z9r5 — APPROVED** [review]

**Pull request merged**